### PR TITLE
Display budgets on the profile

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- point to GUIMAAS
- set up some budgets: https://github.com/juju/juju-gui/pull/1838#issuecomment-231647753
- load your profile
- you should see your budgets listed

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5408/
Test PASSed.

File: jujugui/static/gui/src/app/components/user-profile/user-profile.js [hunk 1]
```diff
@@ -124,6 +129,37 @@ YUI.add('user-profile', function() {
...
+      Callback for the plans API call to get budgets.
+
+      @method _listBudgetsCallback
+      @param {String} error The error from the request, or null.
+      @param {Object} data The data from the request.
+    */
+    _listBudgetsCallback: function(error, data) {
+      this.setState({loadingBudgets: false});
+      if (error) {
+        // TODO frankban: notify the user with the error.
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/user-profile.js]: s/frankban/huwshimi? ;)

File: jujugui/static/gui/src/app/components/user-profile/user-profile.js [hunk 2]
```diff
@@ -683,6 +748,33 @@ YUI.add('user-profile', function() {
     },
 
     /**
+      Generate the header for the budgets.
+
+      @method _generateBudgetHeader
+      @returns {Array} The markup for the header.
+    */
+    _generateBudgetHeader: function() {
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/user-profile/user-profile.js]: I'm sure we can come up with some table components in the future to simplify all of this faux table stuff.

**Comment by hatched** [comment]
:+1: Looks good thanks @huwshimi

**Comment by kadams54** [comment]
:+1: and QA is :ok:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5415/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks @hatched and @kadams54 :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**